backend_new/prisma/migrations/20231213003135_change_to_prisma/migration.sql

```diff
@@ -0,0 +1,1717 @@
+
+-- RenameIndex
+
+ALTER INDEX "UQ_60b3294568b273d896687dea59f" RENAME TO "jurisdictions_name_key";
+
+-- RenameIndex
+
+ALTER INDEX "IDX_94041359df3c1b14c4420808d1" RENAME TO "listing_images_listing_id_idx";
+
+-- RenameIndex
+
+ALTER INDEX "IDX_ba0026e02ecfe91791aed1a481" RENAME TO "listings_jurisdiction_id_idx";
+
+-- RenameIndex
+
+ALTER INDEX "UQ_df3802ec9c31dd9491e3589378d" RENAME TO "user_accounts_email_key";
+
+-- DropForeignKey
+
+ALTER TABLE "paper_applications"
+DROP CONSTRAINT "paper_applications_file_id_fkey";
+
+-- AddForeignKey
+
+ALTER TABLE "paper_applications" ADD CONSTRAINT "paper_applications_file_id_fkey"
+FOREIGN KEY ("file_id") REFERENCES "assets"("id") ON
+DELETE CASCADE ON
+UPDATE NO ACTION;
+
+-- Re-add non-null fields
+ -- AlterTable
+
+ALTER TABLE "application_flagged_set"
+ALTER COLUMN "rule"
+SET NOT NULL;
+
+-- AlterTable
+
+ALTER TABLE "applications"
+ALTER COLUMN "status"
+SET NOT NULL,
+ALTER COLUMN "submission_type"
+SET NOT NULL;
+
+-- AlterTable
+
+ALTER TABLE "translations"
+ALTER COLUMN "language"
+SET NOT NULL;
+
+-- AlterTable
+
+ALTER TABLE "unit_rent_types"
+ALTER COLUMN "name"
+SET NOT NULL;
+
+-- AlterTable
+
+ALTER TABLE "unit_types"
+ALTER COLUMN "name"
+SET NOT NULL;
+
+
+ALTER TABLE "generated_listing_translations"
+ALTER COLUMN "language"
+SET NOT NULL;
+
+-- AlterTable
+
+ALTER TABLE "paper_applications"
+ALTER COLUMN "language"
+SET NOT NULL;
+
+-- The following don't exist on prod DBs but in our 0_init and the /backend/core/src/migration/1675268723433-migration-reset.ts file
+-- -- AlterTable
+-- ALTER TABLE "translations" RENAME CONSTRAINT "PK_aca248c72ae1fb2390f1bf4cd87" TO "translations_pkey";
+-- -- RenameIndex
+-- ALTER INDEX "REL_5eb038a51b9cd6872359a687b1" RENAME TO "alternate_contact_mailing_address_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_194d0fca275b8661a56e486cb6" RENAME TO "applications_applicant_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_3a4c71bc34dce9f6c196f11093" RENAME TO "applications_accessibility_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_56abaa378952856aaccc64d7eb" RENAME TO "applications_alternate_contact_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_7fc41f89f22ca59ffceab5da80" RENAME TO "applications_alternate_address_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_b72ba26ebc88981f441b30fe3c" RENAME TO "applications_mailing_address_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_fed5da45b7b4dafd9f025a37dd" RENAME TO "applications_demographics_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_61b80a947c9db249548ba3c73a" RENAME TO "listings_utilities_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_ac59a58a02199c57a588f04583" RENAME TO "listings_features_id_key";
+-- -- RenameIndex
+-- ALTER INDEX "REL_4ca3d4c823e6bd5149ecaad363" RENAME TO "units_ami_chart_override_id_key";
+ -- The following is in our prod HBA db but not everywhere else
+-- ALTER TABLE
+--     "user_roles" DROP CONSTRAINT "UQ_87b8888186ca9769c960e926870";
```

Comment:
This was my doing on the commenting out, but these can be removed now that it is in the new HBA migration

Reply:
removed! Resolving!